dynamo/plot/utils.py

```diff
@@ -162,14 +169,15 @@ def _matplotlib_points(
     sym_c=False,
     inset_dict={},
     show_colorbar=True,
+    projection=None,  # default in matplotlib
     **kwargs,
 ):
     import matplotlib.pyplot as plt
     from matplotlib.ticker import MaxNLocator
 
     dpi = plt.rcParams["figure.dpi"]
     width, height = width * dpi, height * dpi
-
+    rasterized = kwargs["rasterized"] if "rasterized" in kwargs.keys() else None
```

Comment:
if cell number > 2500, we may automatically set rasterized=True

Reply:
See old file line 277. It is just a code equivalent refactor to keep all “rasterized" variables in the same place.

Reply:
I was suggesting to use rasterize=True when we have a lot of cells. This can save the space